<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="showinfo_tableView">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>249</y>
      <width>661</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>661</width>
      <height>191</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="a">
     <attribute name="title">
      <string>execl表格</string>
     </attribute>
     <widget class="QPushButton" name="start_btn">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>130</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>开始替换</string>
      </property>
     </widget>
     <widget class="QLabel" name="seg_label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>60</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>分隔符：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="xlsxpath_lineEdit">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>20</y>
        <width>481</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="export_btn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>导出数据</string>
      </property>
     </widget>
     <widget class="QLabel" name="xlsxpath_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>70</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>execl文件：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="xlsxpath_btn">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>选择</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="seg_lineEdit_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>70</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>_</string>
      </property>
     </widget>
     <widget class="QComboBox" name="mark_cBox_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="mark_label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>标识项：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>自定义</string>
     </attribute>
     <widget class="QPushButton" name="startTwo_btn">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>开始替换</string>
      </property>
     </widget>
     <widget class="QComboBox" name="item_cBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="item_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>添加项：</string>
      </property>
     </widget>
     <widget class="QLabel" name="input_label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>标签：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_lineEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="seg_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>分隔符：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="seg_lineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>_</string>
      </property>
     </widget>
     <widget class="QLabel" name="mark_label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>保留项：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="mark_cBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="add_btn">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
     <widget class="QLabel" name="target_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>60</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>目标格式：</string>
      </property>
     </widget>
     <widget class="QLabel" name="prompt_label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(14, 0, 255);</string>
      </property>
      <property name="text">
       <string>(0为不保留)</string>
      </property>
     </widget>
     <widget class="QTableView" name="showitem_tableView">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="prompt_label_3">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(14, 0, 255);</string>
      </property>
      <property name="text">
       <string>(右键删除！)</string>
      </property>
     </widget>
     <widget class="QLabel" name="showname_label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>221</width>
        <height>27</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid gray;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="len_label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>70</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>自增量位数：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="len_cBox">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="target_label_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string> 格式表：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exportTwo_btn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>导出数据</string>
      </property>
     </widget>
     <widget class="QComboBox" name="extension_cBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>75</width>
        <height>27</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>.jpg</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="extension_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>扩展名：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTreeView" name="path_treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>211</width>
      <height>550</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid gray;</string>
    </property>
    <property name="text">
     <string>选择图片路径：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="workpath_lineEdit">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>20</y>
      <width>591</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="workpath_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>70</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>图片路径：</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
